### ISSUE TITLE
Main rules notes

### ISSUE THREAD

**Issue by IJW-Wartrader** [GameBrains/er-core#34]
As discussed via video, a place to list items we need to look at in the core rules.

### Orders and conditions > Immobilised condition

This is currently listed as a detachment condition that prevents orders and movement.

**Issue:** It's a unit-level condition rather than a detachment-level condition, and the original GW rules for Immobilised and War Engine orders lead to _weird_ interactions like using March orders, getting Immobilised in the first part of the Shooting phase, and then being able to fire at full effect in the second part because you're no longer applying the March order!

**Suggestion:** Rewrite as a unit-level rule that prevents movement and affects being shot at, but doesn't affect orders. Presumably move to the War Engine rules, maybe as a partial so that it can be used in the movement, critical damage, and shooting at war engines page.

**Comment by digi-brain** [comment]
It'd definitely good to start collating these things. However, we really ought to have a separate issue for each thing to keep things easier to manage (imagine if we have five or ten items in this thread and then we need to discuss some of them in depth...that will get unwieldy in this thread, fast). As per GitHub documentation:

>Breaking a large issue into smaller issues makes the work more manageable and enables team members to work in parallel. It also leads to smaller pull requests, which are easier to review.

>To track how smaller issues fit into the larger goal, use task lists, milestones, or labels. For more information, see "[About task lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/about-task-lists)", "[About milestones](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones)", and "[Managing labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels)."

I'm not 100% sure which method is best in this case — but it seems like this is sort of a 'parent issue' under which we'd like to collate/discuss/track sub-issues, and it sounds like the task lists approach would facilitate that. (See the link above.) So I suggest we try that.

You don't have to create the issue first — you can create your list in your post at the head of this issue, and then convert list items into related issues as required: [Creating an issue from a task list item](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-task-list-item)

**Comment by digi-brain** [comment]
> **Suggestion:** Rewrite as a unit-level rule that prevents movement and affects being shot at, but doesn't affect orders. Presumably move to the War Engine rules, maybe as a partial so that it can be used in the movement, critical damage, and shooting at war engines page.

I agree in principle, but an idea for a slightly different take: What if we just tweak the existing 'Conditions' section a bit? I mean, move the substance of the first paragraph:

>When a detachment becomes subject to a condition then any orders that were in effect are no longer valid — remove the order marker (if any) and place the appropriate condition marker.

...into the explanation for the Broken condition.

And then make it plain that the Immobilised condition only applies to individual units. The "remove the order marker" part will no longer apply here because we'll have moved that instruction so that it is specific to the Broken condition. (Of course, we can make it plain that existing orders are still in force and that Immobilised is **additional** for the affect unit(s), to be crystal clear.)

**Comment by IJW-Wartrader** [comment]
As per the video discussion, this was planned as a place to list of things we need to check in the future. So a task list sounds good.

On the specific issue, I have a counterpoint. Immobilised is not a detachment level rule, it doesn't affect* orders, and it's specific to War Engines. Why would it be in the Main rules section instead of the War Engine section?

*Other than making it pointless to use movement-based orders.

**Comment by digi-brain** [comment]
> On the specific issue, I have a counterpoint. Immobilised is not a detachment level rule, it doesn't affect* orders, and it's specific to War Engines. Why would it be in the Main rules section instead of the War Engine section?
> 
> *Other than making it pointless to use movement-based orders.

Note that this topic is in **summary** form with a consistent structure for each item — I thought it was nice to summarise the effects that relate to all six faces of the order die, all on the same page. Where there is more to say, detailed explanations exist in pertinent topics elsewhere (as in 'Broken detachments'). The bit that makes it odd is that for some orders/conditions there really isn't much else to say outside of the summary...

I see what you mean about Immobilised though. So perhaps so other arrangement would be better. My idea above was just an idea. Go with your idea if you think it's better than having it all on one page. Or, I have a feeling that a third way might emerge when we play around with it.

---

Also, an aside in relation to conditions: I think we could actually view *Flak* and *Preparatory Bombardment* as unit-specific conditions, rather than 'orders' in their own right, somewhat like the Immobilised condition. I mean, you declare one set of Special orders which can combine both Flak and Preparatory Bombardment, to apply to all appropriate **individual** units in the detachment — and we treat the rest of the detachment as though they are on Overwatch orders. One order in force with a single marker, with a 'treat as Overwatch' order effect, and two conditions in force for specific individual units as appropriate while the Special orders are in effect.

I'm not sure whether or not this might actually be a useful distinction though — it's just a half-baked thought at the moment.
